fritz.defaults.yaml

```diff
@@ -143,11 +177,11 @@ skyportal:
     WeatherWidget:
       resizeable: false
       layouts:
-        xlg: [ 0, 10, 4, 1 ]
-        lg: [ 0, 10, 4, 1 ]
-        md: [ 0, 10, 4, 1 ]
-        sm: [ 0, 10, 4, 1 ]
-        xs: [ 0, 12, 4, 1 ]
+        xlg: [0, 10, 4, 1]
```

Comment:
@acrellin was also reporting weird linting behavior. Might be worth looking into why this is happening.

Reply:
Yea I ran `./fritz lint` to make sure and everything passed